conf/application.conf

```diff
@@ -6,10 +6,18 @@ play.crypto.secret="Replace me, please!"
 
 panda.system="editorial-production-metrics"
 
-play.evolutions.autoApply=true
-
 user = ${?USER}
 
+slick.dbs.default.profile="slick.jdbc.PostgresProfile$"
+slick.dbs.default.db.driver="org.postgresql.Driver"
+slick.dbs.default.db.url="jdbc:postgresql://localhost:5903/metricsdb"
+slick.dbs.default.db.user="postgres"
+slick.dbs.default.db.password="postgres"
```

Comment:
Have these come out of dynamo and the local config file?

Reply:
No, they are coming from the `build.sbt` file where we are starting up docker and postgres. They are just random values used in tests only.